src/pages/login/index.js

```diff
@@ -38,6 +67,7 @@ const Login = () => {
             text="Log in"
             onClick={() => onLogin(formData.email, formData.password)}
             classes="green width-full"
+            disabled={!emailIsValid && !passwordIsValid}
           />
         </div>
```

Comment:
you should also add a red label / text around the form field, or the entire form explaining to the user that the email is not valid or the form is not valid